pkg/system/phase2_creating.go

```diff
@@ -511,6 +511,16 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &configMapVolumeMounts)
 			}
+
+			// We are mounting the log path inside 'noobaa-log-processor' container here
```

Comment:
noobaa-log-processor won't be using this PVC as it is using the none-persistent solution -  move it to core container instead.

Reply:
Done, Thanks